internal/services/video_service.go

```diff
@@ -108,23 +111,23 @@ func (v *VideoService) Encode() error {
 func (v *VideoService) Finish() error {
 	err := os.Remove(os.Getenv("localStoragePath") + "/" + v.Video.ID + ".mp4")
```

Comment:
mapping this environment variable to a variable in internal would be interesting, think about it, you have to rename the environment variable with another name